package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "@ewerk/null-or-undefined",
-  "version": "1.1.1",
+  "version": "1.2.2",
```

Comment:
In the npm-publish workflow is something missing.
ToDo: Commit and push version change.